en/messages.json

```diff
@@ -351,5 +351,66 @@
   },
   "integrateContainers": {
     "message": "Integrate your Containers with Mozilla VPN"
+  },
+  "backupFailure": {
+    "message": "Something goes wrong, backup failed: $errorMessage$",
+    "description": "(Options menu) When exporting containers as a file failed",
+    "placeholders": {
+      "errorMessage": {
+        "content": "$1"
+      }
+    }
+  },
+  "containersRestored": {
+    "message": "$restoredCount$ containers restored.",
```

Comment:
Webextension doesn't support proper plurals, we should stay away from using this type of form (it would fail even for English with 1. Something along the line of `Containers restored: $restoredCount$` would scale.